packages/terra-table/src/TableRows.jsx

```diff
@@ -35,17 +36,35 @@ function cloneChildItems(children, onClick, onKeyDown) {
     if (child.type === TableRow) {
       return React.cloneElement(child, newProps);
     }
+    if (child.type === TableSubheader) {
+      return React.cloneElement(child, { count: numberOfCols });
```

Comment:
I believe you'll want this to set `colspan` instead of `count`. Currently there is an error rendering in the console related to `count`:

`warning.js:36 Warning: Unknown prop `count` on <td> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop`

<img width="1440" alt="screen shot 2017-05-16 at 3 19 27 pm" src="https://cloud.githubusercontent.com/assets/633148/26126443/39de8198-3a4b-11e7-87f7-85175f30fab1.png">

The lack of `colspan` also seems to be limiting the `<TableSubheader />` component from spanning the entire table:
<img width="1202" alt="screen shot 2017-05-16 at 3 17 54 pm" src="https://cloud.githubusercontent.com/assets/633148/26126492/64900a88-3a4b-11e7-9e07-4ff33036b2f3.png">